### ISSUE TITLE
Hangouts do not load call section

### ISSUE THREAD

**Issue by miki725** [getferdi/ferdi#335]
**Describe the bug**

Call section in Hangouts never loads. It loads fine in various browsers.

**To Reproduce**

1. Add Hangouts service.
2. Attempt to open the "phone" call section (phone icon).
3. It says "Loading contacts..." which never load.

**Expected behavior**
List of contacts should load.

**Screenshots**
![image](https://user-images.githubusercontent.com/932940/73559154-86d67580-4422-11ea-989e-7e5d553633f2.png)

**Desktop (please complete the following information):**

- OS: [e.g. iOS] Mac OS
- Browser [e.g. chrome, safari] inside Ferdi
- Version [e.g. 22] 5.4.3

**Comment by vantezzen** [comment]
Very weird, there are no errors in the console that are related to this and the other functions seem to work.

**Comment by miki725** [comment]
any idea how to debug further? I dont see anything wrong as well on my end.

**Comment by vantezzen** [comment]
![Screenshot 2020-02-02 at 16 08 57](https://user-images.githubusercontent.com/10333196/73610349-6b13d080-45d6-11ea-8fb9-76deb7795027.png)
![Screenshot 2020-02-02 at 16 07 48](https://user-images.githubusercontent.com/10333196/73610350-6b13d080-45d6-11ea-96bc-bfb994f33d0f.png)

These two requests seem to fail everytime Ferdi tries to make them. I don't use Google Hangouts so when looking at these requests in the browser I don't really see any important data.
![Screenshot 2020-02-02 at 16 13 12](https://user-images.githubusercontent.com/10333196/73610391-ed9c9000-45d6-11ea-9a1e-4e0411b4909a.png)
Could you be able to inspect the second request (to `account?check...`) with in normal Google Chrome of Firefox (somewhere where it works) and look if there the data in the request looks like it could be the list for the calling panel?

**Comment by miki725** [comment]
hmm Im seeing something different.

* on both chrome and ferdi I dont see any call to `api.js`.

* on chrome `account?...` I see a little more output although not much and it seems some overall metadata call like whats the price of calls per minute type of thing. also both calls return same payload:

![image](https://user-images.githubusercontent.com/932940/73611101-69cdae00-45ac-11ea-9327-14a2726dfe87.png)

* on ferdi I dont see `account?...` call at all. basically on ferdi when I press the phone icon, there is zero activity in network tools.

* on ferdi only I see failing is actually version check of ferdi itself:

![image](https://user-images.githubusercontent.com/932940/73611126-aef1e000-45ac-11ea-83f9-c0f89b5a24eb.png)

this is so odd. it literally loads fine everywhere else...

**Comment by vantezzen** [comment]
In Ferdi you'll have to go to the service developer tools in order to see network activity from Google services - they are separated.
To open them,, go to "View" > "Toggle service developer tools"

**Comment by miki725** [comment]
cool. thanks. yep there seeing same error as you are. I even tried uninstalling and reinstalling from scratch in case it was some corrupt config or something. same issue. wonder if its some bug on Google side like user-agent header or something Google does not like. previously I was using Station however recently something changed where Google only allows to make calls from Chrome so I started looking at alternatives. Ferdi seems super nice since I was a previous Franz user until v5. Would love this to work eventually. any more ideas for debugging?

**Comment by miki725** [comment]
and btw thanks for such amazing quick responses 😄

**Comment by vantezzen** [comment]
I am also mostly working on a clean install with the newest recipe version so that can't be the problem.
Does this work on Station?

I originally thought this might be related to the "`Refused to set unsafe header "Connection"`" error in the console but I removed that header using JavaScript breakpoints and it didn't fix the other problems.
I also currently don't have any more debugging ideas as I really don't see any reasons why it would error out like this.

**Comment by miki725** [comment]
Yes it works in Station except cant make calls but hangouts fully loads.

**Comment by 2ec0b4** [comment]
Hello,

Just download Ferdi and got a "Couldn't sign you in" message when trying to log in to Hangouts. I think it is related to this behavior.

> This browser or app may not be secure. [Learn more](https://support.google.com/accounts/answer/7675428?hl=en-GB)
> Try using a different browser. If you’re already using a supported browser, you can refresh your screen and try again to sign in.

Hope this will help you.

**Comment by vantezzen** [comment]
@2ec0b4 This shouldn't be related to the Hangouts error - the error you described should already be fixed with Ferdi 5.4.4-beta.1 but the Hangouts error persists.

**Comment by 2ec0b4** [comment]
@vantezzen you're right: I just downloaded Ferdi 5.4.4-beta.1 and i was able to connect to Hangouts (after updating the service). But i don't have the error here, the contact list loads well

**Comment by vantezzen** [comment]
Does the phone/call section load as well?

**Comment by 2ec0b4** [comment]
- Contacts: OK
- Conversations: OK
- Phone calls: doesn't seem OK

**Comment by 2ec0b4** [comment]
Sorry, i thought the error was on all tabs because i got a similar issue with Franz (This is why i just downloaded Ferdi :) )
